mpls-over-udp-ip/draft-ietf-detnet-mpls-over-udp-ip.xml

```diff
@@ -360,42 +363,14 @@
    &rfc4385;
    &rfc7510;
    <?rfc include="reference.RFC.8174"?>
-   <reference anchor='I-D.ietf-detnet-mpls'>
-     <front>
-       <title>DetNet MPLS
-       </title>
-       <author>
-         <organization>Korhonen, J., Varga, B.</organization>
-       </author>
-       <date year='2019' />
-     </front>
-   </reference>
-   <reference anchor='I-D.ietf-detnet-ip'>
-     <front>
-       <title>DetNet IP
-       </title>
-       <author>
-         <organization>Korhonen, J., Varga, B.</organization>
-       </author>
-       <date year='2019' />
-     </front>
-   </reference>
+   <?rfc include="reference.I-D.ietf-detnet-mpls'?>

```

Comment:
Adding a "Procedures" section
Adding a "Management and Control Information Summary"